sparkle/src/components/Tabs.tsx

```diff
@@ -56,7 +58,8 @@ const TabsTrigger = React.forwardRef<
       <TabsPrimitive.Trigger
         ref={ref}
         className={cn(
-          "s-border-0 s-border-b-2 s-border-primary-800/0 s-pb-1 disabled:s-pointer-events-none data-[state=active]:s-border-primary-800",
+          "s-h-[43px]",
```

Comment:
Can we avoid arbitrary value? Where is this one coming from?

Reply:
don't like it either ✅ 🤫